jmclient/jmclient/wallet_utils.py

```diff
@@ -376,9 +376,14 @@ def wallet_showutxos(wallet_service, showprivkey):
             key = wallet_service.get_key_from_addr(av['address'])
             tries = podle.get_podle_tries(u, key, max_tries)
             tries_remaining = max(0, max_tries - tries)
+            try:
+                mixdepth = wallet_service.wallet.get_details(av['path'])[0]
+            except (IndexError, NotImplementedError):
+                mixdepth = -1
```

Comment:
I'm wondering is this try block here really needed at all, as these exceptions means something is totally wrong and probably it's better to just fail here...

Reply:
You're right. Under normal circumstances, no exception should be thrown. Will refactor and resubmit.